modules/price/commerce_price.module

```diff
@@ -26,7 +26,9 @@ function commerce_price_theme() {
  * Implements hook_ENTITY_TYPE_insert() for 'configurable_language'.
  */
 function commerce_price_configurable_language_insert(ConfigurableLanguage $language) {
-  // Allow the importer to import any found translations for the new language.
-  $importer = \Drupal::service('commerce_price.currency_importer');
-  $importer->importTranslations([$language->getId()]);
+  if (!\Drupal::isConfigSyncing()) {
```

Comment:
I'm really not sure what happened here, all that work is from a commit bradjones1 made for issue #2901821. It's the last commit made before my work after rebasing against branch 8.x-2.x. I may have screwed up the rebase but I'm not sure how this would come up as a result of something I did wrong.